CompetitionMars/CompetitionMars/Pages/CertificationPage.cs

```diff
@@ -0,0 +1,258 @@
+
+
+        public void AddCertification(string Certificate, string From, string CertificationYear)
+        {
+
+            Wait.WaitToBeClickable(driver, "XPath", "//*[@data-tab='third']", 15);
+            certificationTab.Click();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+
+            addNewButton.Click();
+
+            certificateTextbox.SendKeys(Certificate);
+            certificateFromTextbox.SendKeys(From);
+            certificateYearDropdown.SendKeys(CertificationYear);
+            
+            addButton.Click();
+            Console.WriteLine("Certificate added");
+            
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+
+        }
+
+        public string GetAddedCertification()
+        {
+            return addedCertificate.Text;
+        }
+
+        public void UpdateCertification(string Certificate, string From, string CertificationYear)
+        {
+            certificationTab.Click();
+
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+
+            //find the row in table
+
+            //IWebElement row = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/table/tbody[1]/tr"));
+
+            // if (row != null) 
+
+            // {
+            updateIcon.Click();
+            certificateTextbox.Clear();
+            certificateTextbox.SendKeys(Certificate);
+
+            certificateFromTextbox.Clear();
+            certificateFromTextbox.SendKeys(From);
+
+            certificateYearDropdown.SendKeys(CertificationYear);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(15);
+            updateButton.Click();
+            //Thread.Sleep(3000);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5); 
+
+            driver.Navigate().Refresh();
+
+
+            // }
+
+            // else 
+            // {
+            //  Console.WriteLine("Not found");
+            //}
+        }
+
+        public string GetUpdatedCertification()
+        {
+            IWebElement updatedCertificate = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[5]/div[1]/div[2]/div/table/tbody/tr/td[1]"));
+            return updatedCertificate.Text;
+
+        }
+
+        public void DeleteCertification()
+        {
+           
+            certificationTab.Click();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+            JsonHelper jsonHelperObject = new JsonHelper();
+            string jsonFilePath = "C:\\internship notes\\CompetitionMars\\CompetitionMars\\CompetitionMars\\CompetitionMars\\TestData\\DeleteCertificateTestData.json";
```

Comment:
load test data from test or test steps, pages should only contain logic around elements